www/src/components/guidelines/color/swatch.js

```diff
@@ -0,0 +1,130 @@
+import React from "react"
+import CopyToClipboard from "react-copy-to-clipboard"
+
+import { Box } from "../system"
+import { SrOnly } from "../typography"
+
+export default class Swatch extends React.Component {
+  state = {
+    value: this.props.color.hex,
+    displayCopied: false,
+  }
+
+  toggleCopied = () => {
+    this.setState({
+      displayCopied: true,
+    })
+
+    setTimeout(() => {
+      this.setState({
+        displayCopied: false,
+      })
+    }, 2500)
+  }
+
+  handleClick = event => {
+    event.preventDefault()
+    event.stopPropagation()
+  }
+
+  render() {
+    const { a11yLabel, color, swatchStyle, textColor } = this.props
+
+    return (
+      <Box
+        bg={color.hex}
+        css={{
+          ...swatchStyle,
+          ":hover > .btn-copy": {
+            display: `block`,
+          },
+        }}
+      >
+        {a11yLabel !== `×` && (
+          <Box
+            color={textColor}
+            fontSize={0}
+            position="absolute"
+            fontWeight={0}
+            css={{
+              bottom: 2,
+              left: 3,
+              lineHeight: 1,
+              top: `auto`,
+            }}
+          >
+            {a11yLabel}
+          </Box>
+        )}
+
+        <CopyToClipboard text={this.state.value} onCopy={this.toggleCopied}>
+          <button
+            className="btn-copy"
+            css={{
+              background: `none`,
+              border: 0,
+              bottom: 0,
+              color: `black`,
+              cursor: `pointer`,
+              height: `100%`,
+              left: 0,
+              position: `absolute`,
+              right: 0,
+              top: 0,
+              width: `100%`,
+              zIndex: 1,
+              ":focus .tooltip, :hover .tooltip": {
+                display: `block`,
+              },
+            }}
+            onClick={this.handleClick}
+          >
+            <Box
+              bg="white"
+              boxShadow="raised"
+              borderRadius={1}
+              fontSize={1}
+              className="tooltip"
+              css={{
+                top: -40,
+                height: 32,
+                left: 0,
+                lineHeight: `32px`,
+                display: `none`,
+                position: `absolute`,
+                width: 80,
+              }}
+            >
+              {this.state.displayCopied ? (
+                `Copied!`
+              ) : (
+                <>
+                  Copy HEX <SrOnly>${color.hex}</SrOnly>
```

Comment:
I think the visual of the hex color could be useful, what do you think?

Reply:
S/th like edadcdd? 🤔 
![image](https://user-images.githubusercontent.com/21834/59544388-70ba8a00-8f11-11e9-8a43-fcd270174cf8.png)